Gordon360/Program.cs

```diff
@@ -29,7 +29,7 @@
 string corsPolicy = "360UI";
 builder.Services.AddCors(p => p.AddPolicy(name: corsPolicy, corsBuilder =>
 {
-    corsBuilder.WithOrigins(builder.Configuration.GetValue<string>("AllowedOrigin")).AllowAnyMethod().AllowAnyHeader();
+    corsBuilder.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader();
```

Comment:
Please undo this change. AllowAnyOrigin is useful for local testing but should not be committed to main branches.